menus/context.cson

```diff
@@ -10,7 +10,7 @@
     ]
   ]
 
-  '.tree-view.full-menu .header.list-item':[
+  '.tree-view.full-menu .header.list-item':[ to '.tool-panel.tree-view .header.list-item':[
```

Comment:
Or `.tree-view.full-menu .header.list-item` to `.tree-view .full-menu .header.list-item` also works!
What difference? R: One simple space betwen `.tree-view` and `.full-menu` class declaration.